mod/gc_group_layout/views/default/groups/profile/summary.php

```diff
@@ -52,26 +55,42 @@
 		<div class="groups-info col-xs-10 col-md-10 ">
             <h1 class="group-title">
                 <?php
-               
-                    echo gc_explode_translation($group->name, $lang);
 
+                	// cyu - check to see if the user is a gsa-crawler, then display in (both) languages
+                	if (strstr(strtolower($_SERVER['HTTP_USER_AGENT']), 'gsa-crawler') !== false) {
+                		
+                		$json_name = json_decode($group->name,true);
+                		if (json_last_error() === JSON_ERROR_NONE)
+                			$group_name = "{$json_name['en']} / {$json_name['fr']}";
+                		else
+                			$group_name = $group->name;
+
+                	} else {
+
+		                $group_name = ($group->title3) ? gc_explode_translation($group->title3, $lang) : $group->name;
```

Comment:
i left the code on line 70 as is (did not delete, just modified the code as single liner)